common/stringFunctions.lua

```diff
@@ -78,4 +78,81 @@ end
 -- print(string.partition("blaksjdfsaldkj","ldkj"))
 -- print(string.partition("blaksjdfsaldkj","aks"))
 
+if not string.formatSI then
+	local mathFloor = math.floor
+	local mathLog10 = math.log10
+	local mathPow = math.pow
+	local stringFormat = string.format
```

Comment:
This localization is pointless since it's being done inside the function, they get redefined every single time the function is called, so aren't saving any global table lookups.

Reply:
It isn't done inside the function. The function definition starts in line 122.